test/hemera/plugin.async-await.js

```diff
@@ -66,7 +66,48 @@ describe('Async await Plugin interface', function() {
     })
   })
 
-  it('Should be able to register an array of plugins', function(done) {
+  it('Should able to run async await callback function', function (done) {
+    const nats = require('nats').connect(authUrl)
+
+    const hemera = new Hemera(nats)
+
+    // Plugin
+    function plugin(hemera, options) {
```

Comment:
Wrong plugin signature. The callback is never called. Use `async` or `next` callback.

```js
    async function plugin(hemera, options) {
      hemera.add(
        {
          topic: 'math',
          cmd: 'add'
        },
        async req => {
          console.log('#########')
          const result = await Promise.resolve({
            result: req.a + req.b
          })
          return result
        }
      )

      return await true
    }
```